docs/UsersGuide/best/hub-pattern.md

```diff
@@ -17,4 +17,134 @@ ARINC 653 Channels, High-speed hardware buses between nodes, and UARTs between n
 ## Generic Hub
 
 There is now a standard implementation of the hub pattern. The [GenericHub](../api/c++/html/svc_generic_hub_component.html) is an
-implementation of the hub pattern that passes through F´ ports and `Fw::Buffer`s.
\ No newline at end of file
+implementation of the hub pattern that passes through F´ ports and `Fw::Buffer`s.
+
+## Detailed Example Generic Hub Setup
+
+A possible usage of the Hub pattern includes running F´ on multiple computers acting as one F´ system. This example could 
+serve as a base for a multi-computer system running F´.
+
+![Detailed Hub](../media/detailed_hub_example.png)
+
+In this example, a basic implementation of the [Communication Adapter Interface](../../Design/
+communication-adapter-interface.md) 
+facilitates the transfer of serialized data between the two nodes across two different deployments. This interface 
+may be reinforced by including a `Fw::ComStub` and `Fw::ComQueue`, which are excluded in this implementation for 
```

Comment:
Reinforced can have several meanings and I'm wondering if it could confuse folks. What about phrasing the sentence like this?

While they are excluded from this example for simplicity, `Fw::ComStub` and `Fw::ComQueue` components could be added to make the communication link between hubs more robust.